src/CMakeLists.txt

```diff
@@ -236,12 +236,20 @@ ADD_FEATURE_INFO(ntlmclient GIT_NTLM "NTLM authentication support for Unix")
 # Optional external dependency: libgssapi
 IF (USE_GSSAPI)
 	FIND_PACKAGE(GSSAPI)
+	IF (APPLE)
+		INCLUDE(FindGSSFramework)
+	ENDIF()
 ENDIF()
 IF (GSSAPI_FOUND)
 	SET(GIT_GSSAPI 1)
 	LIST(APPEND LIBGIT2_LIBS ${GSSAPI_LIBRARIES})
+	ADD_FEATURE_INFO(SPNEGO GIT_GSSAPI "SPNEGO authentication support")
+ENDIF()
+IF (GSSFRAMEWORK_FOUND)
+	SET(GIT_GSSAPI 1)
+	LIST(APPEND LIBGIT2_LIBS ${GSSFRAMEWORK_LIBRARIES})
+	ADD_FEATURE_INFO(SPNEGO GIT_GSSAPI "SPNEGO authentication support (fwk)")
```

Comment:
What is fwk?  Framework?  I think more verbosity here is better, eg `SPNEGO authentication support (macOS GSS.Framework)`

Reply:
True, I wrote that kinda not expecting such quick success. I'll reword.